src/cloudsc_loki/CMakeLists.txt

```diff
@@ -416,9 +416,65 @@ if( HAVE_CLOUDSC_LOKI )
         OMP 1
     )
 
-    ##############################################################################
-    # C-transpilation mode for generating vectorized C host code (experimental!)
-    ##############################################################################
+    ####################################################
+    ##  SCC CUF (CUDA Fortran)                        ##
+    ##   * SCC with CUDA Fortran (CUF)                ##
+    ##   * allows for 3 types/kind of transformation  ##
+    ##    * parametrised (dimensions) local arrays    ##
+    ##    * host side hoisted local arrays            ##
+    ##    * dynamic memory allocation for local       ##
+    ##      arrays (not recommended)                  ##
+    ####################################################
+
+    cloudsc_xmod( loki-scc-cuf )
+
+    loki_transform_convert(
+        MODE cuf FRONTEND ${LOKI_FRONTEND} CPP
+        CONFIG ${CMAKE_CURRENT_SOURCE_DIR}/cloudsc_cuf_loki.config
+        PATH ${CMAKE_CURRENT_SOURCE_DIR}
+        HEADERS
+            ${COMMON_MODULE}/yomcst.F90
+            ${COMMON_MODULE}/yomphyder.F90
+            ${COMMON_MODULE}/yoethf.F90
+            ${COMMON_MODULE}/yoecldp.F90
+        INCLUDES ${COMMON_INCLUDE}
+        DEFINITIONS CLOUDSC_GPU_TIMING
+        DATA_OFFLOAD REMOVE_OPENMP
+        XMOD ${_TARGET_XMOD_DIR} ${XMOD_DIR}
+        OUTPATH ${CMAKE_CURRENT_BINARY_DIR}/loki-scc-cuf
+        TRAFOTYPE ${TRAFOTYPE}
```

Comment:
`TRAFOTYPE` should be filled with a default value. Otherwise build will fail if it is not set explicitly (e.g. when using CMake/ecbuild without ecbundle). But wait for the discussion on this API on the Loki-PR.